src/locale/locales/nb-NO.json

```diff
@@ -135,6 +135,7 @@
     "min": "Må være minst {0} tegn",
     "min_value": "Må være {0} eller mer",
     "numeric": "Kan bare inneholde nummer",
+    "number": "Må være et gyldig nummer",
```

Comment:
Lurer på om "tall" blir mer presist/naturlig? 0.5 er for eksempel et tall, men ikke et nummer ;)